lineman/app/components/email_settings_page/email_settings_page.scss

```diff
@@ -0,0 +1,17 @@
+.email-settings-page {
+  max-width: $main-max-width;
+  margin: 0px auto;
+}
+
+.email-settings-page__email-settings {
```

Comment:
I'm kind of surprised we don't have a .lmo-card general purpose card class. Maybe this could be the first one?